my-app/src/Nav.js

```diff
@@ -23,10 +23,10 @@ const NavBoard = () => (
     <div class="navbar-collapse collapse dual-collapse">
       <ul class="navbar-nav ml-auto">
         <li class="nav-item">
-          <Link to="/" class="nav-link">Link</Link>
+          <Link to="/profile" class="nav-link">Profile</Link>
```

Comment:
Not sure what to do about this, but I edited this part of Nav.js in the "backend" branch, in order to accommodate checking whether the user is logged in or not. I suppose we can approve this for now and then overwrite it when we merge "backend" to master?